src/gss_sec_ctx.c

```diff
@@ -1289,6 +1312,8 @@ uint32_t gssntlm_inquire_sec_context_by_oid(uint32_t *minor_status,
         return gssntlm_spnego_req_mic(minor_status, ctx, data_set);
     } else if (gss_oid_equal(desired_object, &sasl_ssf_oid)){
         return gssntlm_sasl_ssf(minor_status, ctx, data_set);
+    } else if (gss_oid_equal(desired_object, &sspi_session_key_oid)) {
```

Comment:
Shouldn't this be changed to use GSS_C_INQ_SSPI_SESSION_KEY?

Reply:
done.